packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1236,6 +1236,16 @@ abstract class RangeSliderValueIndicatorShape {
   /// width because it is derived from a formatted string.
   Size getPreferredSize(bool isEnabled, bool isDiscrete, { TextPainter labelPainter });
 
+  /// Determines the best offset to keep this shape on the screen.
```

Comment:
If this is intended to be overridden, maybe say so in the comment.

Reply:
Done